app/templates/settings/labs.hbs

```diff
@@ -157,15 +157,15 @@
                                 {{else if this.redirectFailure}}
                                     {{svg-jar "retry"}} Upload Failed
                                 {{else}}
-                                    Upload redirects JSON
+                                    Upload redirects JSON/YAML
```

Comment:
@peterzimon we are adding second file format (`.yaml`) support here, needs a copy check.

Reply:
My thinking is, if our preferred format is YAML, maybe we should hide the `JSON` option completely and still support it if somebody tries uploading a `.json` file.

Reply:
> My thinking is, if our preferred format is YAML, maybe we should hide the JSON option completely and still support it if somebody tries uploading a .json file.

I like this idea. since YAML is our preferred format we could use `Upload redirects YAML/JSON` or even `Upload redirects YAML` as a button label?

Reply:
I think completely removing `JSON` from the button might confuse users because they would think JSON is not supported anymore. I think it might be better to remove `JSON` in v4.

Reply:
@peterzimon When you have time, can you review the copy [here](https://github.com/TryGhost/Ghost/pull/12187#discussion_r508144220)?

Reply:
@peterzimon ping about the above :wink: